db/seeds.rb

```diff
@@ -6,14 +6,18 @@
   user.password = PASSWORD
   puts 'ユーザーの初期データインポートに成功しました。'
 end
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+
 Text.delete_all
 ImportCsv.text_data
 Movie.delete_all
-ImportCsv.movie_data
\ No newline at end of file
+ImportCsv.movie_data
+
+ADMIN_EMAIL = 'admin@example.com'
+ADMIN_PASSWORD = 'password'
+PASSWORD_CONFIRMATION = 'password'
+
+AdminUser.find_or_create_by!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') do |admin_user|
```

Comment:
`find_or_create_by` の使い方が間違っていますよ。引数に入れるのは，あくまで検索条件です。

```suggestion
AdminUser.find_or_create_by!(email: ADMIN_EMAIL) do |admin_user|
```